app/src/main/java/org/mozilla/fenix/widget/SearchWidgetProvider.kt

```diff
@@ -140,6 +145,7 @@ class SearchWidgetProvider : AppWidgetProvider() {
                 R.layout.search_widget_large -> {
                     setOnClickPendingIntent(R.id.button_search_widget_new_tab, textSearchIntent)
                     setOnClickPendingIntent(R.id.button_search_widget_voice, voiceSearchIntent)
+                    setOnClickPendingIntent(R.id.button_search_widget_new_tab_icon, textSearchIntent)
```

Comment:
A lot of this PR deals with this. Consider changing the name of this PR/commit.

Reply:
Actually, this is a small part of the PR, and it is mentioned in the commit message. If you have another suggestion for the commit message I am open to suggestions.